opendevin/llm/llm.py

```diff
@@ -207,6 +207,25 @@ def get_token_count(self, messages):
         """
         return litellm.token_counter(model=self.model_name, messages=messages)
 
+    def is_local(self):
+        """
+        Determines if the system is using a locally running LLM.
+
+        Returns:
+            boolean: True if executing a local model.
+        """
+        if self.base_url is not None:
+            if (
+                'localhost' not in self.base_url
+                and '127.0.0.1' not in self.base_url
+                and '0.0.0.0' not in self.base_url
+            ):
+                return True
+        elif self.model_name is not None:
```

Comment:
Is it possible to run a local model and _not_ set base_url?

Reply:
`api_base` is passed
https://litellm.vercel.app/docs/providers/ollama
https://litellm.vercel.app/docs/providers/openai_compatible

Reply:
api_base is not required, I tested with both ollama and openai model names and if the base_url is not provided it defaults to localhost and openai respectively